core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -78,18 +75,23 @@
 
   /** If the stream is closed, this can only go from false to true. */
   protected boolean mClosed;
-  /** Whether or not the current block should be cached. */
-  protected boolean mShouldCacheCurrentBlock;
-  /** Include incomplete blocks if Alluxio is configured to store blocks in Alluxio storage. */
-  private final boolean mShouldCachePartiallyReadBlock;
- /** Current position of the stream. */
+  /**
+   * Current position of the file instream.
+   */
   protected long mPos;
+
+  /** Include partially read blocks if Alluxio is configured to store blocks in Alluxio storage. */
```

Comment:
`if Alluxio is configured to store blocks in Alluxio storage` is a bit confusing, do you mean if this stream is configured to cache read blocks?